app/src/lang/translations/en-US.yaml

```diff
@@ -2372,3 +2372,49 @@ export_dashboard: Export Dashboard
 duplicate_dashboard: Duplicate Dashboard
 search_dashboard: Search Dashboard...
 search_flow: Search Flow...
+onboarding:
+  privacy_note:
+    We take user privacy very seriously. We'll never sell or share your information with anyone outside of our
+    organization.
+  action:
+    skip: Skip
+    first: Start Setup
+    continue: Continue
+    saveAndContinue: Save and Continue
+  welcome:
+    title: Welcome to Directus!
+    text: |
+      Whether you're starting from scratch, a seasoned pro, or just checking out Directus for the first time, we're so thrilled you're here. 
+
+      To get your project up and running as quickly as possible, the next few pages will guide you through initial user and project set-up. Ready? Let's get started!
+
+      By the way – Be sure to join our [Directus Discord](https://directus.chat) to get updates, community support, and to see what awesome stuff other users are building.
+  project:
+    title: Let's setup your project
+    text: |
+      We built Directus to be completely customizable. From URL, to logo, to color scheme, you can give your project a distinct identity to make it truly *yours*.
```

Comment:
This is a bit of general issue but while it's very obvious what logo and color scheme are about, it's not clear what the URL is for. I had to look up myself in the docs to be sure [^1].

In that sense, it feels a bit wrong to mention the URL in this intro text (at least not without further clarification). One could easily think that's the URL of the Directus instance itself - "Why do I even need to enter that? Not a particularly exciting customization..."

Some ideas for improvements:
- Add a hint to the field
- Not mention it in this intro text (e.g. mention the "project name" instead)
- Clarify in the intro text
- Link the docs (probably not 😅)

[^1]: https://docs.directus.io/user-guide/settings/project-settings.html#general

Reply:
We can actually just change this 

"From colors to logo, you can give your project a distinct identity to..."

Reply:
I like it 👍